packages/pouchdb-replication/src/index.js

```diff
@@ -4,6 +4,24 @@ import sync from './sync';
 function replication(PouchDB) {
   PouchDB.replicate = replicate;
   PouchDB.sync = sync;
+
+  Object.defineProperty(PouchDB.prototype, 'replicate', {
```

Comment:
Whats the reason for using `defineProperty` over just a plain property definition? I would lean towards the simple version if there wasnt a particular advantage, but may be missing something